quick_start.adoc

```diff
@@ -1,22 +1,142 @@
 [#getting-started]
 = Getting started
 
-* <<manage-clusters,Manage clusters>>
-* <<manage-applications,Manage applications>>
-* <<manage-security,Manage security>>
-* <<observability-quick, Observe clusters>>
+* <<introduction,Introduction>>
+* <<create-and-manage-clusters,Create clusters>>
+* <<manifestwork-example, ManifestWork example>>
 
 [#introduction]
 == Introduction
 
 Coming: Architecture and intro.
 
-Before you install multicluster engine for Kubernetes, review the system configuration requirements and settings at link:../install/requirements.adoc#requirements-and-recommendations[Requirements and recommendations]. With a supported version of OpenShift Container Platform installed and running on your hub cluster, you can proceed with link:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online].
+Before you install multicluster engine for Kubernetes, review the system configuration requirements and settings at link:../requirements.adoc#requirements-and-recommendations[Requirements and recommendations]. With a supported version of OpenShift Container Platform installed and running on your multicluster engine for Kubernetes cluster, you can proceed with link:../install_connected.adoc#installing-while-connected-online[Installing while connected online].
 
+[#create-and-manage-clusters]
+== Create and manage clusters
 
-[#manage-clusters]
-== Manage clusters
+You are now ready to create and import clusters.
```

Comment:
Sug:  "...create and import..." -> "...create, import and manage...".  (The managing is the important end-goal, so I think it should be mentioned.  If it weren't for the lack of the cluster-deployment-import support, we could have lessened the prominence of the import step in this getting started.)